drivers/ieee802154/Kconfig.nrf5

```diff
@@ -42,4 +42,18 @@ config IEEE802154_NRF5_EXT_IRQ_MGMT
 	  the system. One example of external radio IRQ provider could be
 	  a radio arbiter used in dynamic multiprotocol applications.
 
+config IEEE802154_NRF5_VENDOR_OUI
+	int "Vendor Organizationally Unique Identifier"
+	default 16043574
+	help
+	  Custom vendor OUI for OpenThread,
+	  which makes 24 oldest bits of MAC address
```

Comment:
What does `oldest` mean here, do  you mean `lowest` instead?

Reply:
Sorry I think I literally copied meaning from Polish language and I didn't realize that it's incomprehensive. I meant 24 most-significant bits and i fixed it.